src/test/java/com/cronutils/model/time/generator/ExecutionTimeUnixIntegrationTest.java

```diff
@@ -36,9 +36,7 @@
 import com.cronutils.model.time.ExecutionTime;
 import com.cronutils.parser.CronParser;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
can we not use `*` notation and call out each import as previously written?

Reply:
sure, fixed